h5py/tests/test_slicing.py

```diff
@@ -238,7 +238,7 @@ def test_slice_zero_length_dimension(self):
         """ Slice a dataset with a zero in its shape vector
             along the zero-length dimension """
         for i, shape in enumerate([(0,), (0, 3), (0, 2, 1)]):
-            dset = self.f.create_dataset('x%d'%i, shape, dtype=np.int, maxshape=(None,)*len(shape))
+            dset = self.f.create_dataset('x%d'%i, shape, dtype=int, maxshape=(None,)*len(shape))
```

Comment:
I think we want `np.int64` here, but I don't its going to make too much difference either way.